### DISCUSSION TITLE
Compute outdoor thermal comfort at city scale

### DISCUSSION THREAD

**Discussion by j3r3m1** [UMEP-dev/UMEP#678] [Q&A]
I suppose this is a question that has been raised several times and dealt by many of you. [Here](https://github.com/UMEP-dev/UMEP/issues/671#issuecomment-2524057814) is the original post.

> I would like to ask one more thing. I am trying to make the UTCI raster map using "Outdoor Thermal Comfort: Spatial Thermal Comfort" for a whole city.
So, for now I have successfully arrived to produce the Mean Radiant Temperature rasters using the "Outdoor Thermal Comfort: SOLWEIG"
Now the only thing that is missing before launching the "Outdoor Thermal Comfort: Spatial Thermal Comfort" is the wind raster.
The problem is that my PC is not able to produce the Wind raster for the whole city at once. So, I was thinking to separate the city on portions, calculate the Wind Raster for each portion and then merge the Wind Rasters into one big raster. Since, I have produced at once the Mean Radiant Temperature for the whole city and the Wind raster in portiond that have been merged , will the "Outdoor Thermal Comfort: Spatial Thermal Comfort" work?
Thanks in advance.

**Comment by j3r3m1** [comment]
The wind and radiant temperature rasters should have the same size, location and resolution. Thus it is strongly recommended to use URock based on the Solweig raster (that in your case you should probably split into several tiles).

@biglimp any other recommandations ?

**Reply by biglimp** [reply]
Yes, you should divide the city in smaller sections to run URock. Also consider an overlap between areas so that you can get a seamless aggregate map when you mosaic you data in the end.

**Reply by AlexandraStankulova** [reply]
But does this mean that i would be able to perform the Spatial Thermal Comfort analysis with a mosaic raster file for the Wind?

**Reply by biglimp** [reply]
Yes, if you allign the mosaic raster to your tmrt map.

**Reply by AlexandraStankulova** [reply]
Ok, thank you so much. I will continue working and try to make it.
Thank you so much for your help.
